src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -64,7 +70,13 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
             );
             return { data: null, error: { type: "noMatchingClient", logId: logId } };
         }
-        dataWithNonNullClients.push({ ...parcel, client: parcel.client });
+
+        let labelCount: number | undefined;
+        if (parcel.events && parcel.events.length > 0 && parcel.events[0].event_data) {
+            labelCount = Number.parseInt(parcel.events[0].event_data);
+        }
+
+        dataWithNonNullClients.push({ ...parcel, client: parcel.client, labelCount: labelCount });
```

Comment:
If the `event_data` doesn't exist, does this get a 0, null, undefined or something else? And does that work ok with everything (looks like the type of this is undefined, `labelCount` in `ParcelsForDelivery` is optional, and `numberOfLabels` is null)
Might be worth normalising the types between all of these

Reply:
It gets an undefined currently. I agree these should be normalised so I've made labelCount non-optional and given it a default value of 0 to simplify things (I think having 0 number of labels makes more sense than having null number of labels). The behaviour should stay the same!